app/controllers/api/v1x0.rb

```diff
@@ -13,5 +13,6 @@ class EndpointsController < Api::V1::EndpointsController; end
     class GraphqlController < Api::V1::GraphqlController; end
     class SourcesController < Api::V1::SourcesController; end
     class SourceTypesController < Api::V1::SourceTypesController; end
+    class AppMetaDataController < Api::V1::AppMetaDataController; end
```

Comment:
No, we do not. Initially I was doing this in case we wanted to have the field GET-able on all versions, but we have decided against it. Working on removing it now.